CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ BUG FIXES:
 
   * Fix enterprise license application when ACLs are turned off
 
+IMPROVEMENTS:
```

Comment:
We'll add this change when we do a release. Please remove it for now because it causes issues with merging.